threenums.c

```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
+int main() {
+	printf("\nEnter a number:");int num1;
+	scanf("%d",&num1);
+	printf("\nEnter another number:"); int num2;
+	scanf("%d",&num2);
+	printf("\nEnter a last number:"); int num3;
+	scanf("%d",&num3);
+	if (num1==num2 && num2==num3)
+		printf("Numbers are equal.\n");
+	else
```

Comment:
Since you didn't include curly braces for the else statement, the code automatically executed the portion where you print the highest number, which should only be done in the case where the numbers aren't equal.